### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by Thovi98** [comment]
@ericgaspar I think I’ve fixed the issue here, if you want to take a look.

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: scripts/_common.sh [hunk 1]
```diff
@@ -4,11 +4,11 @@
 # COMMON VARIABLES
 #=================================================
 
-#REMOVEME? YNH_PHP_VERSION="8.1"
+phpversion="8.2"
```
Comment (alexAubin) [inline scripts/_common.sh:7]:
```suggestion
```

(There should be no need to define this variable, it's already a setting defined in the core during the apt resource)

**Comment by Thovi98** [comment]
I'd suggest reverting to nodejs 18 for the `Cannot find module 'moment-locales-webpack-plugin'`, who knows 🥲

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Pull request merged**